docs/4.3/openssh_teleport.md

```diff
@@ -0,0 +1,311 @@
+
+Lastly, configure the OpenSSH client to use the Teleport proxy when connecting
+to nodes with matching names. Edit `~/.ssh/config` for your user or
+`/etc/ssh/ssh_config` for global changes:
+
+``` bash
+# work.example.com is the jump host (proxy). credentials will be obtained from the
+# openssh agent.
+Host work.example.com
+    HostName 192.168.1.2
+    Port 3023
+
+# connect to nodes in the work.example.com cluster through the jump
+# host (proxy) using the same. credentials will be obtained from the
+# openssh agent.
+Host *.work.example.com
+    HostName %h
+    Port 3022
+    ProxyCommand ssh -p 3023 %r@work.example.com -s proxy:%h:%p
+
+# when connecting to a node within a trusted cluster with name "remote-cluster",
+# add the name of the cluster to the invocation of the proxy subsystem.
+Host *.remote-cluster.example.com
+   HostName %h
+   Port 3022
+   ProxyCommand ssh -p 3023 %r@work.example.com -s proxy:%h:%p@remote-cluster
+```
+
+When everything is configured properly, you can use ssh to connect to any node
+behind `work.example.com` :
+
+``` bash
+$ ssh root@database.work.example.com
+```
+
+!!! tip "Note"
+
+    Teleport uses OpenSSH certificates instead of keys which means
+    you can not connect to a Teleport node by IP address. You have to connect by
+    DNS name. This is because OpenSSH ensures the DNS name of the node you are
+    connecting is listed under the `Principals` section of the OpenSSH
+    certificate to verify you are connecting to the correct node.
+
+
+To connect to the OpenSSH server via `tsh`, add `--port=<ssh port>` with the `tsh ssh` command:
+
+Example ssh to `database.work.example.com` as `root` with a OpenSSH server on port 22 via `tsh`:
+
+```bash
+tsh ssh --port=22 root@database.work.example.com
+```
+
+!!! warning "Warning"
+
+    The principal (username) being used to connect must be listed in the Teleport user/role configuration.
+
+### OpenSSH Rate Limiting
+
+When using a Teleport proxy in "recording mode", be aware of OpenSSH built-in
+rate limiting. On large number of proxy connections you may encounter errors
+like:
+
+``` bash
+channel 0: open failed: connect failed: ssh: handshake failed: EOF
+```
+
+See `MaxStartups` setting in `man sshd_config` . This setting means that by
+default OpenSSH only allows 10 unauthenticated connections at a time and starts
+dropping connections 30% of the time when the number of connections goes over 10
+and when it hits 100 authentication connections, all new connections are
+dropped.
+
+To increase the concurrency level, increase the value to something like
+MaxStartups 50:30:100. This allows 50 concurrent connections and a max of 100.
+
+Teleport is a standards-compliant SSH proxy and it can work in environments with
+existing SSH implementations, such as OpenSSH. This section will cover:
+
+* Configuring OpenSSH client `ssh` to login into nodes inside a Teleport
+  cluster.
+* Configuring OpenSSH server `sshd` to join a Teleport cluster.
```

Comment:
This section looks like it belongs at the top rather than here?

Reply:
moved